cluster-scope/overlays/prod/moc/smaug/kustomization.yaml

```diff
@@ -134,10 +134,6 @@ resources:
   - ../../../../base/storage.k8s.io/storageclasses/ocs-external-storagecluster-ceph-rbd
   - ../../../../base/user.openshift.io/groups/cluster-admins
 
-  - ingresscontrollers/default.yaml
-  - nodenetworkconfigurationpolicies/vlan-210-nfs.yaml
-  - nodenetworkconfigurationpolicies/vlan-211-nese.yaml
```

Comment:
can you re-add these?

Reply:
okay, did a lot of surgery and squashing. I think we're good this time.